README.md

```
@@ -1,17 +1,17 @@
+[![Go Report Card](https://goreportcard.com/badge/github.com/codeready-toolchain/registration-service)](https://goreportcard.com/report/github.com/codeready-toolchain/registration-service)
+[![GoDoc](https://godoc.org/github.com/codeready-toolchain/registration-service?status.png)](https://godoc.org/github.com/codeready-toolchain/registration-service)
+[![Codecov.io](https://codecov.io/gh/codeready-toolchain/registration-service/branch/master/graph/badge.svg)](https://codecov.io/gh/codeready-toolchain/registration-service)
+[![Operator CD](https://github.com/codeready-toolchain/registration-service/actions/workflows/operator-cd.yml/badge.svg)](https://github.com/codeready-toolchain/registration-service/actions/workflows/operator-cd.yml)
 
 This is the Developer Sandbox Registration Service repository. It implements the registration flow for the Toolchain SaaS.
 
-== Build
+## Build
 
-Requires Go version 1.24.x (1.24.13 or higher) - download for your development environment https://golang.org/dl/[here].
+Requires Go version 1.26.x (1.26.5 or higher) - download for your development environment [here](https://golang.org/dl/).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)